docs/en/user_guides/finetune.md

```diff
@@ -228,3 +228,61 @@ It's because our training schedule is for a batch size of 128. If using 8 GPUs,
 just use `batch_size=16` config in the base config file for every GPU, and the total batch
 size will be 128. But if using one GPU, you need to change it to 128 manually to
 match the training schedule.
+
+## Evaluate the fine-tuned model on ImageNet variants
+
+It's a common practice to evaluate the ImageNet-(1K, 21K) fine-tuned model the ImageNet-1K validation set. This set
+shares similar data distribution with the training set, but in real world, the inference data is more likely to share
+different data distribution with the training set. To have a full evaluation of model's performance on
+out-of-distribution datasets, research community introduces the ImageNet-variant datasets, which shares different data
+distribution with that of ImageNet-(1K, 21K)., MMClassification supports evaluating the fine-tuned model on
+[ImageNet-Adversarial (A)](https://arxiv.org/abs/1907.07174), [ImageNet-Rendition (R)](https://arxiv.org/abs/2006.16241),
+[ImageNet-Corruption (C)](https://arxiv.org/abs/1903.12261), and [ImageNet-Sketch (S)](https://arxiv.org/abs/1905.13549).
+You can follow these steps below to have a try:
+
+### Prepare the datasets
+
+You can download these datasets from [OpenDataLab](https://opendatalab.com/) and refactor these datasets under the
+`data` folder in the following format:
+
+```text
+   imagenet-a
+        ├── meta
+        │   └── val.txt
+        ├── val
+   imagenet-r
+        ├── meta
+        │   └── val.txt
+        ├── val/
+   imagenet-s
+        ├── meta
+        │   └── val.txt
+        ├── val/
+   imagenet-c
+        ├── meta
+        │   └── val.txt
+        ├── val/
+```
+
+`val.txt` is the annotation file, which should have the same style as that of ImageNet-1K. You can refer to
+[prepare_dataset](https://mmclassification.readthedocs.io/en/1.x/user_guides/dataset_prepare.html) to generate the
+annotation file.
+
+### Modify the config file
+
+There are few modifications to the config file, but change the `data_root` of the test dataloader and pass the
+annotation file to the `test_evaluator`.
+
+```python
+# You should replace imagenet-x below with imagenet-c, imagenet-r, imagenet-a
+# or imagenet-s
+test_dataloader=dict(dataset=dict(data_root='data/imagenet-x'))
+test_evaluator=dict(meta_file_path='data/imagenet-x/meta/val.txt')
+```
+
+### Start test
+
+This step is the common test step, you can follow this [guide](https://mmclassification.readthedocs.io/en/1.x/user_guides/train_test.html)
+to evaluate your fine-tuned model on out-of-distribution datasets.
+
+To make it easier, we also provide an off-the-shelf config files, for [ImageNet-C](https://github.com/open-mmlab/mmclassification/tree/dev-1.x/projects/example_project/ood_eval/vit_ood-eval_toy-example.py) and [ImageNet-C](https://github.com/open-mmlab/mmclassification/tree/dev-1.x/projects/example_project/ood_eval/vit_ood-eval_toy-example_imagnet-c.py), and you can have a try.
```

Comment:
The configuration file given by the project is a bit redundant. Examples can be given like this, where the original cfg can be left unchanged. Can be written directly in the doc, no need to give actual examples

```
_base_ = 'mmcls::resnet/resnetv1c50_8xb32_in1k.py'  # can be your own config

test_dataloader=dict(dataset=dict(data_root='data/imagenet-x'))
test_evaluator=dict(meta_file_path='data/imagenet-x/meta/val.txt')
```

Reply:
It's better to keep these configuration files, such that users can try ood evaluation without any efforts. For these contents in these files, I will rewrite them according to your suggestions.